test/npm/util/collaborators.js

```diff
@@ -13,76 +13,60 @@ test('package.name not a string', async t => {
 	);
 });
 
-const npmVersionFixtures = [
-	{version: '9.0.0', accessCommand: 'npm access list collaborators np --json'},
-];
+const accessCommand = (name = 'np') => `npm access list collaborators ${name} --json`;
 
-for (const {version, accessCommand} of npmVersionFixtures) {
-	const npmVersionCommand = {
-		command: 'npm --version',
-		stdout: version,
-	};
+const collaboratorsStdout = stripIndent`
+	{
+		"sindresorhus": "read-write",
+		"samverschueren": "read-write",
+		"itaisteinherz": "read-write"
+	}
+`;
 
-	const collaboratorsStdout = stripIndent`
-		{
-			"sindresorhus": "read-write",
-			"samverschueren": "read-write",
-			"itaisteinherz": "read-write"
-		}
-	`;
+test('main', createFixture, [{
+	command: accessCommand(),
+	stdout: collaboratorsStdout,
+}], async ({t, testedModule: {collaborators}}) => {
+	t.deepEqual(
+		await collaborators({name: 'np'}),
+		collaboratorsStdout,
+	);
+});
 
-	test(`npm v${version}`, createFixture, [
-		npmVersionCommand,
-		{
-			command: accessCommand,
-			stdout: collaboratorsStdout,
+// TODO: this is timing out, seemingly the command isn't matching for Sinon
+// eslint-disable-next-line ava/no-skip-test
+test.skip('external registry', createFixture, [{
+	command: `${accessCommand()} --registry http://my-internal-registry.local`,
+	stdout: collaboratorsStdout,
+}], async ({t, testedModule: {collaborators}}) => {
```

Comment:
This test is behaving weirdly. It's passing, but afterwards it hangs and causes the test file to timeout.

Reply:
AVA 6 changed to no longer force exit the process at the end, which may be related.